apps/koala-frontend/src/app/features/sessions/pages/session/session.page.ts

```diff
@@ -185,6 +185,7 @@ export class SessionPage implements OnInit, OnDestroy {
     this.sessionUpdatedSubscription?.unsubscribe();
     this.toolbarUpdatedSubscription?.unsubscribe();
     this.timerSubscription?.unsubscribe();
+    this.mediaControlService.destroy();
```

Comment:
Why is this necessary? Not sure if we should interfere with angular managed / injected services

Reply:
It destroys the wavesurfer not the actual angular service

Reply:
Without the whole browser crashes when u switch between session and analysis. Took me quite some time to debug this!

Reply:
https://github.com/KoALa-MHF/koala-app/blob/main/apps/koala-frontend/src/app/features/sessions/services/media-control.service.ts#L259